src/device/device_ledger.cpp

```diff
@@ -118,7 +118,19 @@ namespace hw {
     #endif
 
     /* ===================================================================== */
-    /* ===                         Device                               ==== */
+    /* ===                       Internal Helpers                       ==== */
+    /* ===================================================================== */
+    static bool is_fake_view_key(const crypto::secret_key &pub) {
+      for (int i = 0; i<32; i++) {
+        if (pub.data[i] != 0)  {
+          return false;
+        }        
+      }
+      return true;
```

Comment:
For the record, this could also be return pub == crypto::null_skey. Badly chosen parameter name btw.

Reply:
ok